kettle-plugins/hdfs/src/main/java/org/pentaho/big/data/kettle/plugins/hdfs/trans/HadoopFileInputDialog.java

```diff
@@ -1840,6 +1808,18 @@ private void addErrorTab() {
 
   }
 
+  private SelectionAdapter createErrorHandlingSelectionAdapter( Text textWidget ) {
+
+    SelectionAdapterOptions errorOptions = new SelectionAdapterOptions(
+      SelectionOperation.SAVE,
```

Comment:
You should be able to use "SAVE_TO"  state which is used for files that will be save at runtime rather than the save state.

Reply:
Nice catch.  After talking on Slack I decided to use FILE_OR_FOLDER option instead.